pkg/subctl/cmd/gather/resource.go

```diff
@@ -26,27 +27,32 @@ import (
 )
 
 func ResourcesToYAMLFile(info *Info, ofType schema.GroupVersionResource, namespace string) error {
-	list, err := info.DynClient.Resource(ofType).Namespace(namespace).List(metav1.ListOptions{})
+	lists, err := info.DynClient.Resource(ofType).Namespace(namespace).List(metav1.ListOptions{})
 	if err != nil {
 		return errors.WithMessagef(err, "error listing %q", ofType.Resource)
 	}
-
-	path := filepath.Join(info.DirName, info.ClusterName+"-"+ofType.Resource+".yaml")
-	file, err := os.Create(path)
-	if err != nil {
-		return errors.WithMessagef(err, "error opening file %s", path)
+	if len(lists.Items) == 0 {
+		return errors.New(fmt.Sprintf("No config maps found in namespace %s", namespace))
```

Comment:
This should be a status.warning and not status.error

Reply:
We don't even need a warning. If it returns empty, we don't have anything to write and just skip it.

Reply:
yeah we discussed that too and decided that. It may be useful to us to know there were no resources so we proposed to create another log file with such info.